doc/base/classes.xml

```diff
@@ -44482,21 +44495,24 @@
 			<argument index="0" name="base_object" type="Object">
 			</argument>
 			<description>
-				Return true if a given object uses an instance of this script.
+				Returns true if 'base_object' is an instance of this script.
 			</description>
 		</method>
 		<method name="is_tool" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns true if the script is a tool script. A tool script can run in the editor.
 			</description>
 		</method>
 		<method name="reload">
 			<return type="int" enum="Error">
 			</return>
 			<argument index="0" name="keep_state" type="bool" default="false">
+				If true, preserve existing script instances and subclasses.
```

Comment:
Do we support this type of argument description?

Reply:
Probably not yet, but it could be useful for hints in script editor.

Reply:
Yeah, I'm all for this to be supported. It would improve the generated comments for C# methods.